README.md

```diff
@@ -5,86 +5,115 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/canonical/microceph/microceph)](https://goreportcard.com/report/github.com/canonical/microceph/microceph)
 [![Documentation Status](https://readthedocs.com/projects/canonical-microceph/badge/?version=latest)](https://canonical-microceph.readthedocs-hosted.com/en/latest/?badge=latest)
 
-<p align="center">
-<a href="https://snapcraft.io/microceph">MicroCeph</a> is snap-deployed Ceph with built-in clustering.
-</p>
-
 [![Get it from the Snap Store][snap-button]][snap-microceph]
 
-## Table of contents
 
-* [💡 Philosophy](#-philosophy)
-* [🎯 Features](#-features)
-* [⚡️ Quickstart](#%EF%B8%8Fquickstart)
-* [📖 Documentation](#-documentation)
-* [💫 Project & community](#-project--community)
-* [📰 License](#-license)
+MicroCeph is an opinionated orchestration tool for Ceph clusters at all scales.
+It reduces the complexity of deploying and managing clusters by simplifying various operations like, service placement,
+disk administration and remote replication via RESTful APIs and CLI commands.
+
+Available as a snap, MicroCeph is the easiest tool for administrators, developers, and hobbyists to manage clusters.
+
+## Installation
+
+MicroCeph has first-class support as a snap. On snap-ready systems, you can install it on the command line with:
+
+```
+sudo snap install microceph
+```
+
+Disable automatic snap upgrades, so that no unexpected updates change your set-up:
+
+```
+sudo snap refresh --hold microceph
+```
+
+## Basic usage
+
+MicroCeph can deploy a Ceph cluster on a single machine with minimal commands.
+
+First, set up a Ceph cluster on your machine with:
+
+```
+sudo microceph cluster bootstrap
+```
+
+> [!NOTE]  
+> `cluster` is a MicroCeph subcommand for managing associated Ceph clusters.
+
+After setup, add storage to your cluster with:
+
+```
+sudo microceph disk add loop,4G,3
+```
+
+Here, you’ll add three virtual disks (“loop file” disks) of 4 GiB each.
+
+
+Once your cluster is set up and running, you can monitor its status with:
+
+```
+sudo microceph status
+```
+
+Note that there are no spaces between the `disk add` arguments.
```

Comment:
Think this sentence should be moved up right after the `microceph disk add` command